roles/etcd_upgrade/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  author: Jan Chaloupka
+  description:
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 1.9
```

Comment:
We need to update all meta/main.yml and bump all `min_ansible_version` to the same or run ansible's check.